### PR TITLE
Implement `nameOnly` for `listDatabases`

### LINKED ISSUE #301: Implement `nameOnly` for `listCollections` command
`listCollections` command should support `nameOnly` parameter. See references to this issue in the source code.

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #301.

### CODE REVIEW

File: internal/handlers/pg/msg_listdatabases.go [hunk 1]
```diff
@@ -66,27 +64,47 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
...
 			}
 		}
 	}
 
+	if nameOnly {
+		var reply wire.OpMsg
+		err = reply.SetSections(wire.OpMsgSection{
+			Documents: []*types.Document{must.NotFail(types.NewDocument(
+				"databases", databases,
+				"ok", float64(1),
```
Comment (AlekSi) [inline internal/handlers/pg/msg_listdatabases.go:98]: Only "ok" was missing in #519

**Review by seeforschauer — CHANGES_REQUESTED** [review]
how about tests? ^_^

**Comment by AlekSi** [comment]
They exist already: https://github.com/FerretDB/FerretDB/blob/94b7d1025e9597e53d6cce9796507dba05eb8287/integration/commands_administration_test.go#L126-L128

File: internal/handlers/pg/msg_listdatabases.go (hunk 1 above)
Comment (seeforschauer) [reply internal/handlers/pg/msg_listdatabases.go:98]: thank you!

**Comment by seeforschauer** [comment]
Then test must have failed, and they did not. Or?

**Comment by AlekSi** [comment]
`ListDatabaseNames` uses only `name` fields in response documents. We returned did more queries and returned more data than was needed; that was a problem in performance, but that's valid behavior. Now we do fewer queries, but still work the same way as MongoDB.

**Review by seeforschauer — APPROVED** [review]

**Pull request merged**